video/out/drm_atomic.c

```diff
@@ -452,3 +453,140 @@ bool drm_mode_destroy_blob(int fd, struct drm_mode *mode)
 
     return (ret == 0);
 }
+
+static uint32_t get_property_id(int fd, drmModeObjectProperties *props,
+                const char *name)
+{
+    drmModePropertyPtr property;
+    uint32_t i, id = 0;
+
+    for (i = 0; i < props->count_props; i++) {
+        property = drmModeGetProperty(fd, props->props[i]);
+        if (!strcmp(property->name, name))
+            id = property->prop_id;
+        drmModeFreeProperty(property);
+
```

Comment:
The props are fetched in `drm_object_create_properties()`. Perhaps there's a way to reuse instead of re-fetching?